### ISSUE TITLE
Overflow in doctest

### ISSUE THREAD

**Issue by jchodera** [bhmm/bhmm#17]
@franknoe: Can you look at this?
https://travis-ci.org/bhmm/bhmm#L642-L644

```
Doctest: bhmm.estimators.bayesian_sampling.BayesianHMMSampler._sampleHiddenStateTrajectory ... /home/travis/miniconda/envs/_test/lib/python2.7/site-packages/bhmm-0.4.2+17.g5bcde9e-py2.7-linux-x86_64.egg/bhmm/_external/sklearn/mixture/gmm.py:579: RuntimeWarning: overflow encountered in square
  + np.dot(X ** 2, (1.0 / covars).T))
ok
```

Is this a real issue?

**Comment by franknoe** [comment]
I will have a look on this early next week. It might be an issue.

Am 01/08/15 um 21:24 schrieb John Chodera:

> @franknoe https://github.com/franknoe: Can you look at this?
> https://travis-ci.org/bhmm/bhmm#L642-L644
> 
> |Doctest: bhmm.estimators.bayesian_sampling.BayesianHMMSampler._sampleHiddenStateTrajectory ... /home/travis/miniconda/envs/_test/lib/python2.7/site-packages/bhmm-0.4.2+17.g5bcde9e-py2.7-linux-x86_64.egg/bhmm/_external/sklearn/mixture/gmm.py:579: RuntimeWarning: overflow encountered in square
> - np.dot(X *\* 2, (1.0 / covars).T))
>   ok
>   |
> 
> Is this a real issue?
> 
> —
> Reply to this email directly or view it on GitHub 
> https://github.com/bhmm/bhmm/issues/17.

## 

---

Prof. Dr. Frank Noe
Head of Computational Molecular Biology group
Freie Universitaet Berlin

Phone: (+49) (0)30 838 75354
Web:   research.franknoe.de

## Mail:  Arnimallee 6, 14195 Berlin, Germany

**Comment by franknoe** [comment]
Also fixed by #37
